pkg/metrics/controller_test.go

```diff
@@ -289,6 +289,68 @@ func TestControllerGameServersTotal(t *testing.T) {
 	})
 }
 
+func TestControllerFleet_OnDeleting(t *testing.T) {
```

Comment:
```suggestion
func TestControllerFleetOnDeleting(t *testing.T) {
```

I know some people use `_` we don't tend to here, so just consistency.

Also just noticed some snuck in here - oopsie. We should clean that up at a later date.